gradle/java-module.gradle

```diff
@@ -113,19 +113,22 @@ dependencies {
 	if ( JavaVersion.current().isJava9Compatible() ) {
 		// The JDK used to run Gradle is Java 9+, and we assume that that is the same
 		//		JDK for executing tasks
-		compile( libraries.jaxb )
+		compile( libraries.jaxb_api )
+		compile( libraries.jaxb_runtime )
 		compile( libraries.jaxb_xjc )
 		compile( libraries.jaxb2_basics )
 		compile( libraries.jaxb2_basics_ant )
 		compile( libraries.jsr250_api )
 
-		testCompile( libraries.jaxb )
+		testCompile( libraries.jaxb_api )
+		testCompile( libraries.jaxb_runtime )
 		testCompile( libraries.jaxb_xjc )
 		testCompile( libraries.jaxb2_basics )
 		testCompile( libraries.jaxb2_basics_ant )
 		testCompile( libraries.jsr250_api )
 
-		testRuntime( libraries.jaxb )
+		testRuntime( libraries.jaxb_api )
+		testRuntime( libraries.jaxb_runtime )
```

Comment:
Doesn't jaxb_runtime depend on jaxb_api transitively?

Reply:
It does, but our code actually depends on the API, not the runtime. Ideally we would not add a dependency to the runtime and we would let the user pick one, but as we know that's not a realistic expectation.

Also, now that I think of it, some applications don't use XML configuration at all, and for these applications it would be possible to exclude the dependency to the runtime while keeping the dependency to the APIs, and everything would work fine. If we only add a dependency to the runtime, they will have to exclude the dependency to the runtime, and add their own dependency to the APIs, which seems less... correct.